### PR TITLE
(WIP) (Modules-2709) Reuse power shell tests

### PR DESCRIPTION
**Pull request by jpogran**
Still working on tests

### CODE REVIEW

File: spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env ruby
+require 'spec_helper'
+require 'puppet/type'
+require 'puppet_x/puppetlabs/powershell_manager'
+
+describe PuppetX::Dsc::PowerShellManager, :if => Puppet::Util::Platform.windows? do
+  before(:each) do
+    @ps = PuppetX::Dsc::PowerShellManager.instance("powershell.exe -NoProfile -NonInteractive -NoLogo -ExecutionPolicy, Bypass -Command -'")
```
Comment (ferventcoder) [inline spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb]: @jpogran I Thought I saw a comma when we were pairing!!!

Comment (ferventcoder) [inline spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb]: Then I thought I was just seeing things.

Comment (ferventcoder) [inline spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb]: @Iristyle saw an extra apostrophe as well.

**Comment by Iristyle** [comment]
#165 contains this test now

**Comment by jpogran** [comment]
This PR is obsolete. Closing

**Pull request closed without merging**